### DISCUSSION TITLE
// eslint-disable-next-line wxml/wx-key

### DISCUSSION THREAD

**Discussion by ABoyCDog** [wxmlfile/eslint-plugin-wxml#42] [Ideas]
the online REPL can't work.
like the "// eslint-disable-next-line wxml/wx-key" or "/** eslint-disable wxml/wx-key */", it can't work in wxml file.

**Comment by iChenLei** [comment]
Hi @ABoyCDog , eslint-plugin-wxml not support disable commnet ! As we know, the comment grammer in `wxml` is `<!-- wxml comment -->`, not `//` or `/** **/`. VSCode(monaco) not support auto insert `<!-- -->` to wxml file. And even you write `<!-- eslint-disable wxml/wx-key -->` , it still not work, because eslint-plugin-wxml not implement this feature. If you have any idea, PR welcome.

**Comment by ABoyCDog** [comment]
all right, thanks anyway
